lib/mix/tasks/tableau.build.ex

```diff
@@ -18,6 +18,7 @@ defmodule Mix.Tasks.Tableau.Build do
     {opts, _argv} = OptionParser.parse!(argv, strict: [out: :string])
 
     out = Keyword.get(opts, :out, "_site")
+    token = Map.put(token, :out, out)
```

Comment:
in #125 i'm also fixing this, so once that merges you can rebase and this change shouldnt be necessary anymore.